apps/mobile-wallet/src/components/NFTThumbnail.tsx

```diff
@@ -82,6 +118,12 @@ const NFTThumbnail = ({ nft, size }: NFTThumbnailProps) => {
                   </AttributesGrid>
                 )}
               </ScreenSection>
+
+              {!isDataUri && (
+                <ScreenSection>
+                  <Button title={t('View full size')} onPress={() => openBrowserAsync(nft.image)} />
```

Comment:
Q: what will happen if the `image` filed is a link to a malicious / fishing website? I have the same problem on the explorer. We can't do much about it, can we?

Reply:
I don't think we can, but I am open to ideas.

Reply:
I don't have any, except for preventing from opening the full size image..